examples/xml-attributes/xml_attributes.bal

```diff
@@ -1,41 +1,33 @@
 import ballerina/io;
+import ballerina/lang.'xml as xmllib;
 
 xmlns "http://ballerina.com/aa" as ns0;
 
 public function main() {
     // Creates an XML element, which has attributes that are bound to a namespace as well as ones that are not.
-    xml x1 = xml `<ns0:book ns0:status="available" count="5"/>`;
+    xmllib:Element x1 = <xmllib:Element> xml `<ns0:book ns0:status="available" count="5"/>`;
     io:println(x1);
 
-    // A single attribute that is bound to a namespace can be accessed using its qualified name. 
-    io:println(x1@[ns0:status]);
+    // A single attribute that is bound to a namespace can be accessed using its qualified name.
+    io:println(x1.ns0:status);
 
-    // An attribute can also be accessed using the string representation of the qualified name. 
-    string? s = x1@["{http://ballerina.com/aa}status"];
-    io:println(s);
-
-    // An attribute that is not bound to a namespace can be accessed using the string representation of the name.
-    io:println(x1@["count"]);
-
-    // Accesses an attribute using a dynamic name.
-    string attributeName = "count";
-    io:println(x1@[attributeName]);
+    // Attribute access expressions are lax typed.
```

Comment:
We don't have any documentation except the spec which explains what lax typing is right? If that's the case, we might either have to explain briefly what lax typing is or just explain what's going on here, without referring to lax typing.